modules/auth/proc-adding-azure-as-an-authentication-provider.adoc

```diff
@@ -0,0 +1,75 @@
+[id='proc-adding-azure-as-an-authentication-provider_{context}']
+= Enabling the Microsoft Azure authentication provider
+
+{product} includes a Microsoft Azure authentication provider that can authenticate users by using OAuth.
+
+.Procedure
+. To allow {product-short} to authenticate with Microsoft Azure, create an OAuth Application in Microsoft Azure.
+
+.. Go to link:https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade[*Azure Portal > App registrations*], and create an *App Registration* for {product-short}.
+
+.. On your *App registration* overview page, add a new *Web platform configuration*, with the configuration:
+
+*Redirect URI*:: Enter the backend authentication URI set in {product-short}:
+For local development::: `pass:[http://localhost:7007/api/auth/microsoft/handler/frame]`
+For a non-local deployment::: `pass:[http://{APP_FQDN}:{APP_BACKEND_PORT}/api/auth/microsoft/handler/frame]`
+*Front-channel logout URL*:: Leave blank.
+*Implicit grant and hybrid flows*:: All unchecked.
+
+.. On the *API permissions* tab, click *Add Permission*, then add the following *Delegated permission* for the
+*Microsoft Graph API*:
+
+`email`:::
+`offline_access`:::
+`openid`:::
+`profile`:::
+`User.Read`:::
+(Optionally)::: Optional custom scopes of the Microsoft Graph API defined in the `app-config.yaml` file.
+
+.. (Optionally) Your company might require you to grant admin consent for these permissions.
+Even if your company does not require link:https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/user-admin-consent-overview[admin consent], you might do so as it means users do not need to individually consent the first time they access backstage.
+To grant admin consent, a directory admin must go to this page and click the *Grant admin consent for COMPANY NAME* button.
+
+.. Go to the *Certificates & Secrets* page, then the *Client secrets* tab, and create a new client secret.
+Save the *Secret* for the next step.
+
+. Set up the Microsoft Azure authentication provider in the {product-short} backend (`app-config.yaml`).
++
+Add the `microsoft` provider configuration to your `app-config.yaml` under the root `auth` configuration, and optionally, enable the `gitlab` provider for sign-in:
```

Comment:
```suggestion
Add the `microsoft` provider configuration to your `app-config.yaml` under the root `auth` configuration. Make sure to configure `signInPage: microsoft` to ensure that the frontend displays the appropriate authentication provider.
```
the SignInPage is not optional, without it it can behave unpredictably, if someone configures the auth provider, they mostly like want to set the `signInPage` as well.

Feel free to change what I suggested. The main point is that it should mention that signInPage needs to be set